demo/helm/rook-operator/README.md

```diff
@@ -0,0 +1,110 @@
+rook-operator
+=============
+
+Installs [rook-operator](https://github.com/rook/rook) to create/configure/manage Rook clusters atop Kubernetes.
+
+TL;DR;
```

Comment:
The README was just a search and replace from some of the other charts. I'd not noticed that but will update